packages/mobile/src/components/check-box/check-box.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import { default as RNCheckBox } from '@react-native-community/checkbox';
+import { useState } from '~/hooks/hooks';
+import { ColorPalette } from '@vse-bude/shared';
+
+type Props = {
+  onChange: (value: boolean) => void;
+};
+
+const CheckBox: FC<Props> = ({ onChange }) => {
+  const [checked, setChecked] = useState(false);
+
+  const onValueChange = (newValue: boolean) => {
+    onChange(newValue);
+    setChecked(newValue);
+  };
```

Comment:
```suggestion
const CheckBox: FC<Props> = ({ value, onChange }) => {
```
Actually you can just reexport default component and use it